metals/src/main/scala/scala/meta/internal/metals/debug/DebugProtocol.scala

```diff
@@ -135,6 +144,36 @@ object DebugProtocol {
     }
   }
 
+  object ConfigurationDone {
+    def unapply(
+        request: DebugRequestMessage
+    ): Option[Option[dap.ConfigurationDoneArguments]] = {
+      if (request.getMethod != "configurationDone") None
+      else
+        Some(
+          parse[dap.ConfigurationDoneArguments](request.getParams).toOption
+        )
+    }
+  }
+
+  object DisconnectRequest {
+    def unapply(request: DebugRequestMessage): Option[DisconnectArguments] = {
+      if (request.getMethod != "disconnect") None
```

Comment:
Super nit, but we check against "disconnect" in a couple places. Might want to just make these variables somewhere and use them wherever we need them.